test/module/irohad/ametsuchi/block_query_test.cpp

```diff
@@ -161,63 +150,6 @@ TEST_F(BlockQueryTest, GetZeroBlock) {
       [](auto &&e) { SUCCEED(); });
 }
 
-/**
- * @given block store with 2 blocks totally containing 3 txs created by
- * user1@test AND 1 tx created by user2@test. Block #1 is filled with trash data
- * (NOT JSON).
- * @when read block #1
- * @then get no blocks
- */
-TEST_F(BlockQueryTest, GetBlockButItIsNotJSON) {
```

Comment:
while we keep flat file block storage, we must keep the tests

Reply:
FlatFile could be only used for a persistent storage. For a temporary one Postgres is an only option.

Reply:
The suggestion is to keep the test as it was, and create another task to make a test for `ProtoBlockJsonConverter`.